docs/contributing/managing-pull-requests.md

```diff
@@ -0,0 +1,178 @@
+
+`gatsby-dev-cli` is a command-line tool for local Gatsby development. When making changes in gatsby packages, this helps copy changes in the packages to a Gatsby site that you can test your changes on.
+
+Learn more about `gatsby-dev-cli` [here](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-dev-cli)!
+
+### Commit and PR Title
+
+It's good to have descriptive commit messages so that other folks can make sense of what your commit is doing. We like [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.3).
+
+However, PRs get squashed when merged into our repository so individual commit messages are not as important as PR titles.
+
+Let's look at some examples of good and bad PR titles:
+
+#### Good PR Titles ✅
+
+- chore(docs): Fix links in contributing page
+- feat(gatsby): Add support for per page manifests
+- fix(gatsby-plugin-sharp): Ensure images exist before attempting conversion
+
+#### Bad PR Titles ❌
+
+- new tests
+- add support for my new cms
+- fix bug in gatsby
+
+## Giving Feedback
+
+- Be _nice_. We're stronger everyday because of our community so empathy is important and we want our contributors to feel welcome.
+- Make suggestions using [GitHub's suggestions feature](https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) if possible. This makes accepting your suggestions easier for the author.
+- Link to examples when necessary
+- Try not to [bikeshed](http://bikeshed.com/) too much
+- Note when a suggestion is optional (as opposed to required)
+- Be objective and limit nitpicks (a few are fine if they add value or improve code readability)
+
+## Rights and Permissions
+
+### Who can review a PR?
+
+If you're a member of the [gatsbyjs](http://github.com/gatsbyjs) organisation on GitHub, you can review a PR.
+
+> 💡 Not a member yet? Love us and want to help? Make your first contribution and get invited!
+
+### Who can approve a PR?
+
+Every PR opened in the repository needs to be approved before it can be merged. While anyone who is a member of the [gatsbyjs](http://github.com/gatsbyjs) organisation can approve a PR, to be merged in, it needs to be reviewed by a member of the team that owns that part of Gatsby.
+
+Typically this is:
+
+- **gatsbyjs/core** for Code
+- **gatsbyjs/docs** for Documentation
+
+We also have `CODEOWNERS` set on different parts of the repo and an approval by someone in the `CODEOWNERS` for the file(s) the PR is changing can also suffice.
+
+### Who can merge a PR?
+
+PRs can only be merged by members of the core team and Gatsbot.
+
+#### Gatsbot
+
+Gatsbot is our little android friend that automatically merges PRs that are ready to go. If a PR is approved and all checks are passing, add the `bot: merge on green` label and Gatsbot will merge it in automatically!
+
+## Gotchas
+
+- You might not always be able to push to an author's fork (depending on their settings). In such a case, leave suggestions.
```

Comment:
Maybe explain this in a bit more detail or drop it completely?

Reply:
Could drop the Gotchas section completely?

Reply:
Updated with more detail!